MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -505,6 +506,28 @@ - (void)testWrapperCrashCallback {
   XCTAssertTrue(first.count == 1);
 }
 
+- (void)testAbstractErrorLogSerialization {
+  MSAbstractErrorLog *log = [MSAbstractErrorLog new];
```

Comment:
You probably want to test the value is NO after serialization even though you don't assign any value to log.fatal.